VMEncryption/EV2/ExtensionInfo.Public.xml

```diff
@@ -0,0 +1,31 @@
+<ExtensionInfo>
+    <Namespace>Microsoft.Azure.Security</Namespace>
+    <Type>AzureDiskEncryptionForLinux</Type>
+    <ExtensionShortName>AdeLinux</ExtensionShortName>
```

Comment:
Suggest changing "AdeLinux" to "AzureDiskEncryptionForLinux" if that is short enough of a name for consistency